Refit.Tests/RequestBuilder.cs

```diff
@@ -264,6 +265,13 @@ public class ComplexQueryObject
 
         public string TestAlias2 { get; set; }
 
+        [JsonPropertyName("json-property")]
+        public string TestJsonPropertyName { get; set; }
+
+        [AliasAs("test-alias")]
+        [JsonPropertyName("test-json-property")]
+        public string TestAliAsWithJsonProperty { get; set; }
```

Comment:
Is that a incorrect capital A in mistake with AliAs

Reply:
@glennawatson fixed